platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/action/OrientationChangeAction.java

```diff
@@ -0,0 +1,66 @@
+package com.mapbox.mapboxsdk.testapp.action;
+
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.ContextWrapper;
+import android.content.pm.ActivityInfo;
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+import android.view.ViewGroup;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isRoot;
+
+public class OrientationChangeAction implements ViewAction {
+
+  private final int orientation;
+
+  private OrientationChangeAction(int orientation) {
+    this.orientation = orientation;
+  }
+
+  public static ViewAction orientationLandscape() {
+    return new OrientationChangeAction(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+  }
+
+  public static ViewAction orientationPortrait() {
+    return new OrientationChangeAction(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+  }
+
```

Comment:
While we are at it, can we include `ActivityInfo.SCREEN_ORIENTATION_REVERSE_PORTRAIT` and `ActivityInfo.SCREEN_ORIENTATION_REVERSE_LANDSCAPE` as well?